### PR TITLE
B 18448 int deprecate personally procured moves fields revised

### PR DESCRIPTION
**Pull request by joeydoyecaci**
B 18448

Summary
Removed PersonallyProcuredMoves code from ShipmentSummaryWorksheet in order to be able to remove PersonallyProcuredMoves from Move object. Due to extensive modifications needed to ShipmentSummaryWorksheet, this component will need to be tested thoroughly.

##Testing Steps

Verify all related Unit tests pass (Changes were all Go, but could indirectly impact JS).
As a Customer, Create a PPM under a new move.
Approve it as a Service Counselor
When signing the shipment as Customer verify that all fields are being populated, and no errors occur

### CODE REVIEW

**Review by paulstonebraker — APPROVED** [review]

File: pkg/models/ppm_shipment.go [hunk 1]
```diff
@@ -38,6 +38,8 @@ type PPMCloseout struct {
 type PPMShipmentStatus string
 
 const (
+	// PPMShipmentStatusDraft captures enum value "DRAFT"
```
Comment (pambecker) [inline pkg/models/ppm_shipment.go:41]: is this maybe a copy and paste mistake?  should it be Cancelled?

File: pkg/testdatagen/make_personally_procured_move.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ func MakePPM(db *pop.Connection, assertions Assertions) models.PersonallyProcure
 	mustCreate(db, &ppm, assertions.Stub)
 
 	// Add the ppm we just created to the move.ppm array
-	ppm.Move.PersonallyProcuredMoves = append(ppm.Move.PersonallyProcuredMoves, ppm)
+	//ppm.Move.PersonallyProcuredMoves = append(ppm.Move.PersonallyProcuredMoves, ppm)
```
Comment (pambecker) [inline pkg/testdatagen/make_personally_procured_move.go:40]: Is there a reason this is just commented out and not deleted?

File: pkg/services/shipment_summary_worksheet/shipment_summary_worksheet.go
Comment (antgmann) [inline pkg/services/shipment_summary_worksheet/shipment_summary_worksheet.go:1]: I have confirmed that these changes, and the changes to the SSW test file, do not impede the generation of the SSW PDF. Well done!

**Review by antgmann — COMMENTED** [review]
Please update this branch with IntegrationTesting before merging. Besides that and the comments Pam has made, everything looks good to me!

File: pkg/models/ppm_shipment.go (hunk 1 above)
Comment (joeydoyecaci) [reply pkg/models/ppm_shipment.go]: Comment has been corrected to Cancelled.

File: pkg/testdatagen/make_personally_procured_move.go (hunk 2 above)
Comment (joeydoyecaci) [reply pkg/testdatagen/make_personally_procured_move.go]: This has been removed

File: pkg/services/shipment_summary_worksheet/shipment_summary_worksheet.go
Comment (joeydoyecaci) [reply pkg/services/shipment_summary_worksheet/shipment_summary_worksheet.go:1]: Thank you!

**Review by pambecker — APPROVED** [review]

**Review by antgmann — APPROVED** [review]

**Pull request merged**